### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by Kasi72** [pkjmesra/PKScreener#98] [Ideas]
Thanks for your updates..May I suggest a screener...Big successful trader like "Stock bee" recommends for screening stock price which crosses 4% up in a day for finding and entering before a big move..He also tells about stocks which cross 9 million in volume in a day before making big moves.. I would request you to add a screener  with Both 4% up+9 million volume break out scanner for 1 day time line.

**Comment by pkjmesra** [comment]
@Kasi72 

This is easy. I just made some changes and have pushed the code. Go ahead, download the latest release from https://github.com/pkjmesra/PKScreener/releases or you can simple Check for Update from within your app and download it.

For now you can filter using the defined keys in the filters section of user config file. 

For example, to only show those stocks that have changed by at least 4% and have volume more than 9 million, you can change the config (in the filters sections) to:
```
    minimumchangepercentage = 4
    minimumvolume = 9000000 
```
Save the config file (pkscreener.ini) and then re-launch the app. 🥳 and run the Volume scanner (X > 12 > 9) with whatever volume ratio (compared to 20-SMA of volume) you'd like.

**Reply by pkjmesra** [reply]
https://github.com/pkjmesra/PKScreener/issues/99 created/resolved.
